lib/idp/constants.rb

```diff
@@ -16,6 +16,9 @@ module Vendors
       MOCK = 'mock'
       USPS = 'usps'
       AAMVA = 'aamva'
+      AAMVA_UNSUPPORTED_JURISDICTION = 'UnsupportedJurisdiction'
+      STATE_ID_MOCK = 'StateIdMock'
+      SOURCE_CHECK = [AAMVA, AAMVA_UNSUPPORTED_JURISDICTION, MOCK, STATE_ID_MOCK].freeze
```

Comment:
A/N: We call `source_check` many things in our specs. Happy to move this somewhere else though.

Reply:
Is `MOCK` actually a valid value here? I'm not sure where that makes its way into the result.

Reply:
I've seen it come up in some of the other feature specs, I can take it out of the list.

Reply:
Funny story (and doesn't impact removing MOCK from SOURCE_CHECK): [Hinz is adding 'mock' for resolution vendors for the Socure work](https://github.com/18F/identity-idp/pull/11479).